app/controllers/contratos_controller.rb

```diff
@@ -11,11 +11,11 @@ def show
       @value_false_2 = Float(Contrato.find(params["id"]).valorInicial)*rand(30)/(rand(30))
       @value_false_3 = Float(Contrato.find(params["id"]).valorInicial)*rand(30)/(rand(30))
       @contratos = Contrato.find(params["id"])
-      array_value = [@value_true.round(0),@value_false_1.round(0),@value_false_2.round(0),@value_false_3.round(0)]
       array_objeto = [@contratos.objeto.capitalize]
-      array_question = [array_objeto, array_value.shuffle, @value_true.round(0)]
-      array_question.flatten!
-      render json:array_question
+
+      @array_question = {"name"=>array_objeto, "val1"=>@value_true.round(0), "val2"=>@value_false_1.round(0), "val3"=>@value_false_2.round(0), "val4"=>@value_false_3.round(0),"true" =>@value_true.round(0)}
```

Comment:
Line is too long. [205/80]<br>Space missing after comma.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.